bundles/org.openhab.binding.modbus.sunspec/src/main/resources/ESH-INF/thing/inverter-channel-types.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="modbus" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="ac-total-current-type">
+		<item-type>Number:ElectricCurrent</item-type>
+		<label>AC Total Current Value</label>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	
+	<channel-type id="ac-power-type">
+		<item-type>Number:Power</item-type>
+		<label>AC Power Value</label>
+		<state readOnly="true" pattern="%d %unit%"/>
+	</channel-type>
+	
+	<channel-type id="ac-frequency-type">
+		<item-type>Number:Frequency</item-type>
+		<label>AC Frequency Value</label>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	
+	<channel-type id="ac-apparent-power-type" advanced="true">
+		<item-type>Number</item-type>
+		<label>AC Apparent Power Value</label>
+		<state readOnly="true" />
+	</channel-type>
+	
+	<channel-type id="ac-reactive-power-type" advanced="true">
+		<item-type>Number</item-type>
+		<label>AC Reactive Power Value</label>
+		<state readOnly="true" />
+	</channel-type>
```

Comment:
I'd imagine these still have units though right?

Reply:
Yes, I will add them later, but they were missing from openhab-core. So I've added them there first: https://github.com/openhab/openhab-core/pull/1347 While they are merged, I don't see these changes published yet, so I have no way to simply build against those at the moment.

Reply:
Unfortunately this will be a breaking change later. Even if not physically correct, did you consider using `W` for the time being? This would alt least allow using the correct item-type and not require re-configuration later.

Reply:
Thanks, this is a good idea. I assigned the Watt type, hope this will make the transition easier.